libcxx/test/libcxx/ranges/range.adaptors/range.stride.view/iterator/increment.pass.cpp

```diff
@@ -0,0 +1,27 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// UNSUPPORTED: c++03, c++11, c++14, c++17, c++20
+// UNSUPPORTED: no-exceptions
```

Comment:
These tests shouldn't be marked as `UNSUPPORTED` when exceptions are disabled. `operator++` should definitely be tested when exceptions are disabled. Why did you have to disable it?

This comment applies to all other tests as well.

Reply:
Left over from a copy/paste that I used when I started building these tests. I had no idea what I was doing so I started with what I could find! I will definitely remove them! Thanks!

Reply:
I believe that this is addressed in 60e0e97524706bf1b108d33ff0bc9eea13ca3e5d.